### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by cobap** [SimFin/simfin#9]
# Bug Report

If you experience errors or bugs, please make a serious effort to solve the
problem yourself before asking here. The problem is quite likely in your own
code and a simple Google search for the error-message may help you solve it.
If it is a problem directly related to simfin, then please search the closed
GitHub issues, because it may already have been answered there.

Please make sure you have the latest simfin package installed by running:

    pip install --upgrade simfin
 
And make sure you have downloaded fresh data-files from the SimFin server
by setting `refresh_days=0` (see example below).

If you still cannot solve the problem and need our help, then please provide
the following.

## Description

Please write a brief description of the bug / error you have experienced.

Getting an error when trying to download simfin data.
![image](https://user-images.githubusercontent.com/6344807/139749668-f96f1848-58ce-4fff-a87f-639004dc8cb1.png) 

## System Details

- Python version: 3.9.2
- Simfin version: 0.8.3
- Other relevant package versions
- Operation System and version: Windows 10
- Computer's CPU, RAM-size, free HD space, etc: Intel, 8GB RAM, +100GB free space
- Other relevant system information

## Code Example

```
import simfin as sf
from simfin.names import *

sf.set_data_dir('./data/')
sf.set_api_key(api_key=API_KEY)

data = sf.load(dataset='income', variant='annual', market='us', index=[TICKER, REPORT_DATE], refresh_days=0)

```

## Result / Error

Please write the full output and error message you received. You can also
indent this text-block by pressing the tab-key to get proper formatting
of the error-message.

![image](https://user-images.githubusercontent.com/6344807/139749908-bee74cd6-31e6-4327-b4f0-ffa9939df895.png)

**Comment by thf24** [comment]
Hm I can't reproduce this error (tried with Python 3.9 on MacOS). From your screenshot it looks like requests.get is throwing the error. Which version of requests are you using?
```
import requests
print(requests.__version__)
```

**Comment by cobap** [comment]
![image](https://user-images.githubusercontent.com/6344807/139841813-41317192-91ee-470e-b64d-70f315230343.png)

I think has nothing to do with Requests

Yesterday I uninstalled the package, today, as soon I got your notification, I tried again and successfully downloaded the data.

Maybe some error during package install? I'm not sure. Or a internal server error? 

Anyway, fell free to close the ticket :D

![image](https://user-images.githubusercontent.com/6344807/139842676-92bb3738-bc3a-4064-8ff2-3b19a6b0a771.png)
